tree/ntuple/v7/inc/ROOT/RColumnElement.hxx

```diff
@@ -340,7 +348,9 @@ public:
 template <typename CppT, typename NarrowT>
 class RColumnElementCastLE : public RColumnElementBase {
 protected:
-   explicit RColumnElementCastLE(std::size_t size) : RColumnElementBase(size) {}
+   explicit RColumnElementCastLE(std::size_t size, std::size_t bitsOnStorage) : RColumnElementBase(size, bitsOnStorage)
```

Comment:
With more than one parameters to the constructor: we can also remove the `explicit`.

Reply:
Actually, looking at the [examples listed here](https://en.cppreference.com/w/cpp/language/explicit), it still makes a difference for `copy-list-initialization`... I guess we don't want that either, ie keep the `explicit`?

Reply:
Ok, let's merge like this, we can still remove the `explicit` later on.